### PR TITLE
Review python 2 deprecation notice text

### LINKED ISSUE #911: Review Python deprecation notice
https://docs.conan.io/en/latest/installation.html#python-2-deprecation-notice

The notice seems difficult to read with many repetitions and may be confusing for users. I think it will benefit from a review.

### PR DESCRIPTION
**Pull request by danimtb**
closes #911

### CODE REVIEW

File: installation.rst [hunk 1]
```diff
@@ -165,15 +165,11 @@ The upgrade shouldn't affect the installed packages or cache information. If the
...
-access to that feature. Such features will be clearly documented in code and documentation.
-
-If and when Conan 2.x is released (Not expected in 2018) the level of compatibility with
-Python 2 may be reduced further.
+
+All features of Conan until version 1.6 are fully supported in both Python 2 and Python 3. However, new features in upcoming Conan releases
+that are only available in Python 3 or more easily available in Python 3 will be implemented and tested only in Python 3, and versions of
+Conan using Python 2 will not have access to that feature. This will be clearly described in code and documentation.
+
+If and when Conan 2.x is released (Not expected in 2018) the level of compatibility with Python 2 may be reduced further.
```
Comment (jgsogo) [inline installation.rst]: Shall we consider adding something in this line: _We encourage you to upgrade to Python 3 as soon as possible. If this is impossible for you or your team, we would like to know so Conan can still be used in your organization._ (and add a mail or some way of contact).

⬆️ @memsharded, feedback, please.

Comment (memsharded) [reply installation.rst]: > If this is impossible for you or your team, we would like to know so Conan can still be used in your organization. 

"If this is impossible for you or your team, we would like to know it. Please give feedback at https://github.com/conan-io/conan/issues/3334,  or info@conan.io"

**Comment by jgsogo** [comment]
@danimtb, update with the last comments and I will merge. Thanks!

File: installation.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ with pip pre-installed. However, if necessary you can install pip by following t
 .. warning::
 
     Python 2 will soon be deprecated by the Python maintainers. It is strongly recommended to use Python 3 with Conan, especially if need to manage non-ascii filenames or file contents.
```
Comment (uilianries) [inline installation.rst]: Should we include https://pythonclock.org/ as link reference? I think is a good link to know how much time Python 2 has.

Comment (danimtb) [reply installation.rst]: Maybe as a link in the warning box of this other section https://docs.conan.io/en/latest/installation.html#install-with-pip-recommended

Comment (jgsogo) [reply installation.rst]: The _pythonclock_ is not related to `pip`, I would add it inside this warning or not add it at all. I'm merging this as we want to close this version, but consider adding it as a different PR.

**Pull request merged**